faq.md

```diff
@@ -0,0 +1,32 @@
+## FAQ questions
+
+- Why use a new ABI? Why not an attribute?
```

Comment:
Formatting nit: I think let's use `###` headings (this also means people can link to the questions)